stob/ad.c

```diff
@@ -378,6 +428,8 @@ static int stob_ad_domain_init(struct m0_stob_type *type,
 	bool                       balloc_inited;
 	int                        rc = 0;
 
```

Comment:
Remove the blank line here, and align the identifier "bcfg" with others.

Reply:
Done